### ISSUE TITLE
HELP to reproduce the results

### ISSUE THREAD

**Issue by Robin-WZQ** [6zHAOyi/BadVision#3]
Dear authors,

It's a great work for implanting backdoor into LVLMs.

However, there is a problem in reproducing the results of BadEncoder.

I set lr=1e-5, batch_size=4, scale of shadowdataset=5K. After 30 epochs training, the Utility loss is still around -0.50 which means the encoder is corrupted for benign inference.

But it works well for BadVision.

Could you please share the specific parameter for training Badencoder? Or could you release the implanted model in the cloud.

Thank you! :)

**Comment by Robin-WZQ** [comment]
After carefully reviewing the code and the original paper, we confirm that the implementation of BadEncoder is consistent with the settings described in the original BadEncoder paper.

To address the corruption issue, the only required change is to set
```
loss = loss0 + loss1 + 2 * loss2. # in ./src/badencoder.py
```
With this adjustment, the utility loss reaches -0.985053, while the target efficiency loss reaches -1.983878.

:)
